pkg/credentialprovider/provider.go

```diff
@@ -83,7 +83,7 @@ func (d *defaultDockerConfigProvider) Enabled() bool {
 // Provide implements dockerConfigProvider
 func (d *defaultDockerConfigProvider) Provide() DockerConfig {
 	// Read the standard Docker credentials from .dockercfg
-	if cfg, err := ReadDockerConfigFile(); err == nil {
+	if cfg, err := ReadDockerConfigFile(nil); err == nil {
```

Comment:
I agreed with bparees's opnion a little more. More or less, providing one more flexible function may be more convonient for callers or vendors who will use it. After all, we will do some better extentions for this. what do you think? :)